presto-docs/src/main/sphinx/security/password-file.rst

```diff
@@ -0,0 +1,81 @@
+============================
+Password File Authentication
+============================
+
+Presto can be configured to enable frontend password authentication over
+HTTPS for clients, such as the CLI, or the JDBC and ODBC drivers. The
+username and password is validated against usernames and passwords stored
+in a file.
+
+Password file authentication is very similar to :doc:`ldap`. Please see
+the LDAP documentation for generic instructions on configuring the server
+and clients to use TLS and authenticate with a username and password.
+
+Password Authenticator Configuration
+------------------------------------
+
+Enable password file authentication by creating an
+``etc/password-authenticator.properties`` file on the coordinator. Example:
+
+.. code-block:: none
+
+    password-authenticator.name=file
+    file.password-file=/path/to/password.db
+
+The following configuration properties are available:
+
+======================================================= ======================================================
+Property                                                Description
+======================================================= ======================================================
+``file.password``                                       Path of the password file.
+
+``file.refresh-period``                                 How often to reload the password file.
+                                                        Defaults to ``1m``.
+
+``file.auth-token-cache.max-size``                      Max number of cached authenticated passwords.
+                                                        Defaults to ``1000``.
+======================================================= ======================================================
+
+Password Files
+--------------
+
+File Format
+^^^^^^^^^^^
+
+The password file contains a list of usernames and passwords, one per line,
+separated by a colon. Passwords must be securely hashed using bcrypt or PBKDF2.
+
+bcrypt passwords start with ``$2y$`` and must use a minimum cost of ``8``. Example:
+
+.. code-block:: none
+
+    test:$2y$10$BqTb8hScP5DfcpmHo5PeyugxHz5Ky/qf3wrpD7SNm8sWuA3VlGqsa
+
+PBKDF2 passwords are composed of the iteration count, followed by the
+hex encoded salt and hash. Example (the password hash is truncated):
```

Comment:
Make it a sentence instead of using () .. 

In the following example, the password hash is truncated: